src/components/NewCommentForm.tsx

```diff
@@ -14,24 +94,37 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': false,
```

Comment:
The 'is-danger' class is always set to false. Consider updating this logic to reflect the actual error state for the name field.